documentation/book/proc-uninstalling.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assemblies:
+//
+// master.adoc
+
+[id='uninstalling-{context}']
+= Uninstalling {ProductName}
+
+This procedure describes how to uninstall {ProductName} and remove resources related to the deployment.
+
+.Prerequisites
+
+In order to perform this procedure, identify resources created specifically for a deployment and referenced from the {ProductName} resource.
+
+Resources might include:
+
+* Secrets (Custom CAs and certificates, Kafka Connect secrets, and other Kafka secrets)
+* Logging `ConfigMaps` (of type `external`)
+
+These are resources:
+
+* Referenced by `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, or `KafkaMirrorMaker` configuration
+* Consumed by the operators
```

Comment:
I think we can remove this "Consumed by the operators", I don't think it adds anything.

Reply:
Removed